airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -70,11 +70,23 @@ def __init__(
         self.concurrent_run_limit = concurrent_run_limit
         self.script_location = script_location
         self.retry_limit = retry_limit
-        self.num_of_dpus = num_of_dpus
         self.s3_bucket = s3_bucket
         self.role_name = iam_role_name
         self.s3_glue_logs = 'logs/glue-logs/'
         self.create_job_kwargs = create_job_kwargs or {}
+        
+        if "WorkerType" in self.create_job_kwargs and "NumberOfWorkers" in self.create_job_kwargs:
```

Comment:
Probably better to extract these `in` checks. It’d make this more readable. It’s currently difficult to visually parse out the different `in` and `not in` checks.

Reply:
Extracted the `in` checks and replaced with Boolean variables.